docs/tutorials/13_quantum_random_access_optimizer.ipynb

```diff
@@ -0,0 +1,854 @@
+{
+ "cells": [
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Quantum Random Access Optimization"
```

Comment:
The way the headings come out in the right sidebar of the docs seems odd. I don't see what might be up in terms of levels, at first sight they seem ok. but take for instance a bunch of the headings after References - the nesting under +/- just seems weird. I would have thought those under references to be all lower level headings ie. reflecting the structure. Maybe its nothing to get too hung up about at this point.

![image](https://github.com/qiskit-community/qiskit-optimization/assets/40241007/ea77f965-f044-4e7d-8536-5e5772ea2044)


![image](https://github.com/qiskit-community/qiskit-optimization/assets/40241007/0cd7ea71-bf53-4c64-b74b-67ed04ae8634)